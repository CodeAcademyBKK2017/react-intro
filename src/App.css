.app{
  display: flex;
  height: 100vh;
  justify-content: center;
  align-items: center;
  flex-direction: column;
}
.made-by{
  position: fixed ;
  bottom: 0;
  right: 0;
}
.app-header{
  display: inline;
}

.App {
  text-align: center;
}

.App-logo {
  animation: App-logo-spin infinite 20s linear;
  height: 80px;
}

.App-header {
  background-color: #222;
  height: 150px;
  padding: 20px;
  color: white;
}

.App-title {
  font-size: 1.5em;
}

.App-intro {
  font-size: large;
}

@keyframes App-logo-spin {
  from { transform: rotate(0deg); }
  to { transform: rotate(360deg); }
}

