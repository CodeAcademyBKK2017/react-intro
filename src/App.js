import React, {Component} from 'react';
import Box from './components/Box/Box.component';
import './App.css';

class App extends Component {
  render () {
    return (
      <div className="App">
        <div className="App-intro">
          <Box/>
        </div>
        <div className="KD">Made By KingDark</div>
      </div>
    );
  }
}

export default App;
