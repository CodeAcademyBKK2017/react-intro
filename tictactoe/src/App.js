import React, { Component } from 'react';
// import logo from './logo.svg';
import './App.css';

import Cell from './components/Cell/Cell.component'
import Row from './components/Row/Row.component'
import Box from './components/Box/Box.component'

class App extends Component {
  render() {
    const boxData = [
      ['o', 'x', 'o'],
      ['o', 'x', 'x'],
      ['x', 'x', 'o']
    ];
    return (
      <div>
        {/* <Cell cellData="O" />
        <Row rowData={['x', 'x']} /> */}
        <Box boxData={boxData} />
      </div>
    );
  }
}

export default App;
