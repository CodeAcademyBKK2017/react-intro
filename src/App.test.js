import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import {shallow} from 'enzyme';
import renderer from 'react-test-renderer';

describe('Button component', () => {

  it('App Snapshot Test', () => {
    // const box = shallow(<Box data={mock} />).instance();
    const tree = renderer.create(<App />);
    expect(tree).toMatchSnapshot();
  });

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<App />, div);
  });
});