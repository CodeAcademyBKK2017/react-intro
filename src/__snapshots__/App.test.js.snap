// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component App Snapshot Test 1`] = `
<div
  className="App"
>
  <div
    className="App-intro"
  >
    <div>
      <div>
        <div
          className="Cell "
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "red",
            }
          }
        >
          
        </div>
        <div
          className="Cell "
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "red",
            }
          }
        >
          
        </div>
        <div
          className="Cell "
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "red",
            }
          }
        >
          
        </div>
      </div>
      <div>
        <div
          className="Cell "
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "red",
            }
          }
        >
          
        </div>
        <div
          className="Cell "
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "red",
            }
          }
        >
          
        </div>
        <div
          className="Cell "
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "red",
            }
          }
        >
          
        </div>
      </div>
      <div>
        <div
          className="Cell "
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "red",
            }
          }
        >
          
        </div>
        <div
          className="Cell "
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "red",
            }
          }
        >
          
        </div>
        <div
          className="Cell "
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "red",
            }
          }
        >
          
        </div>
      </div>
    </div>
  </div>
</div>
`;
